### PR TITLE
Updated readme and reworked example for v5.3

### PR DESCRIPTION
**Pull request by icew1nd**
Based off this article https://medium.com/storybookjs/storybook-5-3-83e114e8797c i updated the readme and example to the specs of Storybook v5.3 🎉

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,14 +4,34 @@
...
+Supports storybook v4, v5 and newer
 
-#### Installation
+## Installation
 ```bash
 yarn add storybook-addon-styled-component-theme --dev
 ```
 
+## Configuration
+Register the addon in .storybook/main.ts
```
Comment (echoulen) [inline README.md]: if `.ts`, please change the sample to `main.ts`

File: example/.storybook/main.js [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    stories:['../stories/**/*.stories.(tsx|mdx)'],
+    addons: ['@storybook/preset-create-react-app','storybook-addon-styled-component-theme/dist/register'],
+  };
```
Comment (echoulen) [inline example/.storybook/main.js]: wired indent

File: example/.storybook/preview.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import {addDecorator} from '@storybook/react';
+import {withThemesProvider} from 'storybook-addon-styled-component-theme';
+
+const defaultTheme = {
+    name: "DEFAULT",
+    backgroundColor: "azure",
+    textColor: "dimgrey",
+    borderRadius: "30px",
+  };
```
Comment (echoulen) [inline example/.storybook/preview.js]: wired indent

File: example/package.json [hunk 4]
```diff
@@ -10,9 +10,13 @@
...
   "devDependencies": {
     "@babel/core": "^7.7.5",
-    "@storybook/addons": "^5.2.8",
-    "@storybook/react": "^5.2.8",
+    "@storybook/addons": "^5.3.13",
+    "@storybook/preset-create-react-app": "^1.5.2",
+    "@storybook/react": "^5.3.13",
     "babel-loader": "^8.0.6",
-    "storybook-addon-styled-component-theme": "link:../"
+    "storybook-addon-styled-component-theme": "1.3.0",
```
Comment (echoulen) [inline example/package.json]: please keep `link:../`

Comment (icew1nd) [reply example/package.json]: Just out of curiosity does this make it point to current version in parent folder? 👀 Didnt seem to want to run out of the box with npm

File: README.md (hunk 1 above)
Comment (icew1nd) [reply README.md]: I changed it to main.js as 5.3 throws errors if main is in typescript and i feel like it doesn't matter too much for the example let me know your thoughts though

**Pull request merged**